libraries/AP_TECS/AP_TECS.cpp

```diff
@@ -746,6 +761,8 @@ void AP_TECS::_update_throttle_with_airspeed(void)
         _throttle_dem = 1.0f;
     } else if (_flags.is_gliding) {
         _throttle_dem = 0.0f;
+    // } else if (_flags.overspeed) {
+    //        _throttle_dem = 0.0f;
```

Comment:
Pin so that we don't forget to amend this.

Reply:
> Pin so that we don't forget to amend this.

Hoping these are awful enough that someone is so revolted they come up with the correct solution.  We looked at the input side and the output side and discussed clamping the energy difference somewhere, but couldn't find something we liked.